apps/forms/app/(workspace)/dashboard/new/page.tsx

```diff
@@ -31,9 +31,9 @@ export default async function OnboardWithNewFormPage({
 
   const supabase = createServerComponentClient(cookieStore);
 
-  const { data } = await supabase.auth.getSession();
+  const { data: auth } = await supabase.auth.getUser();
   // if no auth, sign in, redirect back here
-  if (data.session === null || data.session.user === null) {
```

Comment:
Used this similar logic at https://github.com/gridaco/grida/blob/main/apps/forms/app/(site)/organizations/%5Borganization_name%5D/settings/profile/page.tsx#L38-L40 to alias data and to check !auth.user